spark3/src/main/java/org/apache/iceberg/spark/source/SparkBatchScan.java

```diff
@@ -150,24 +169,39 @@ public PartitionReaderFactory createReaderFactory() {
                 .allMatch(fileScanTask -> fileScanTask.file().format().equals(
                     FileFormat.ORC)));
 
+    boolean hasNoRowFilters =
+        tasks().stream()
+            .allMatch(combinedScanTask -> !combinedScanTask.isDataTask() && combinedScanTask.files()
+                .stream()
+                .allMatch(fileScanTask -> OrcRowFilterUtils.rowFilterFromTask(fileScanTask) == null));
```

Comment:
This code was also not touched by https://github.com/linkedin/iceberg/pull/48 and does not exist in apache/iceberg. Can you remove this?

These are several other changes in the files which are linkedin specific and were not touched by https://github.com/linkedin/iceberg/pull/48

Reply:
@shardulm94 sorry yea, you're correct. I think I messed up while merging. I'll create new PR